pkg/deploy/dev-workspace/dev_workspace.go

```
@@ -228,12 +253,12 @@ func checkWebTerminalSubscription(deployContext *deploy.DeployContext) error {
 		subscription); err != nil {
 
 		if apierrors.IsNotFound(err) {
-			return nil
+			return false, nil
 		}
-		return err
+		return false, err
 	}
 
-	return errors.New("a non matching version of the Dev Workspace operator is already installed")
```

We can't silently return `false, nil`
I believe it must be error or warning